pkg/grapicmd/di/providers.go

```diff
@@ -90,30 +78,30 @@ func ProvideToolRepository(ctx *grapicmd.Ctx, gexCfg *gex.Config) (tool.Reposito
 	return toolRepo, nil
 }
 
-func ProvideProtocWrapper(ctx *grapicmd.Ctx, ui clui.UI, toolRepo tool.Repository) protoc.Wrapper {
+func ProvideProtocWrapper(ctx *grapicmd.Ctx, ui cli.UI, toolRepo tool.Repository) protoc.Wrapper {
 	return protoc.NewWrapper(
 		&ctx.ProtocConfig,
 		ctx.FS,
 		ctx.Execer,
 		ui,
 		toolRepo,
 		ctx.RootDir,
-		ctx.BinDir,
 	)
 }
 
-func ProvideInitializeProjectUsecase(ctx *grapicmd.Ctx, gexCfg *gex.Config, ui clui.UI, generator module.Generator) usecase.InitializeProjectUsecase {
+func ProvideInitializeProjectUsecase(ctx *grapicmd.Ctx, gexCfg *gex.Config, ui cli.UI, generator module.Generator) usecase.InitializeProjectUsecase {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported function ProvideInitializeProjectUsecase should have comment or be unexported